src/main/java/it/gov/pagopa/fdr/to/eventhub/util/CommonUtil.java

```diff
@@ -251,4 +265,64 @@ private boolean sendEventToHub(
       return false;
     }
   }
+
+
+  /**
+   * Send a message to the Event Hub
+   */
+  private boolean sendEventBatchToHub(
+      List<String> jsonPayloads,
+      EventHubProducerClient eventHubClient,
+      FlussoRendicontazione flusso,
+      ExecutionContext context) {
+
+    try {
+
+      //
+      EventDataBatch evhEventBatch = eventHubClient.createBatch();
+      int batchMaxSize = evhEventBatch.getMaxSizeInBytes();
```

Comment:
could be changed if there are performance problems. Otherwise, if the performance is good, keep it!

Reply:
I am referring to the fact of the potential sending of smaller batches

Reply:
Currently, in a test that I've made with 100k payments, it seems that the execution ended in about 7 seconds (UAT environment). But I think that the event send operation is asynchronous and does not impact so much on total execution time, either if batches are smaller or larger.